src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3089,6 +3089,15 @@ internal override IEnumerable<ISymbol> GetSymbolsWithName(string name, SymbolFil
             return new NameSymbolSearcher(this, filter, name, cancellationToken).GetSymbolsWithName();
         }
 
+        /// <summary>
+        /// Returns true if there is an implicit conversion from
+        /// <paramref name="fromType"/> to <paramref name="toType"/>.
+        /// </summary>
+        public override bool IsAssignableTo(ITypeSymbol fromType, ITypeSymbol toType)
```

Comment:
>public override bool IsAssignableTo(ITypeSymbol fromType, ITypeSymbol toType) [](start = 8, length = 77)

Consider moving this method next to the ```ClassifyConversion``` that is calls.

Reply:
I'd prefer to keep it in the region that correctly describes its place in the hierarchy.

---
In reply to: [186801079](https://github.com/dotnet/roslyn/pull/26719#discussion_r186801079) [](ancestors = 186801079)